govcd/catalog_test.go

```diff
@@ -963,3 +963,31 @@ func (vcd *TestVCD) Test_CatalogQueryMediaList(check *C) {
 	// Check that media name is what it should be
 	check.Assert(medias[0].Name, Equals, vcd.config.Media.Media)
 }
+
+// Tests System function UploadMediaImage by checking if provided UDF type standard iso file uploaded.
+func (vcd *TestVCD) Test_CatalogUploadMediaImageWihUdfTypeIso(check *C) {
+	fmt.Printf("Running: %s\n", check.TestName())
+
+	if vcd.config.Media.MediaUdfTypePath == "" {
+		check.Skip("Skipping test because no UDF type iso path given")
```

Comment:
Another message that doesn't make sense. Please fix in some other open PR.
(Why this PR was merged without my approval? ;) )